README.md

```diff
@@ -1,4 +1,4 @@
-[![build status](https://circleci.com/gh/docker/cli.svg?style=shield)](https://circleci.com/gh/docker/cli/tree/master) [![Build Status](https://jenkins.dockerproject.org/job/docker/job/cli/job/master/badge/icon)](https://jenkins.dockerproject.org/job/docker/job/cli/job/master/)
```

Comment:
Thanks! Looks like we forgot to update after moving to a new Jenkins cluster. I had a quick look, but looks like the new Jenkins gives a 404 for that URL; https://ci.docker.com/public/job/cli/job/master/badge/icon


I _suspect_ it's missing a plugin to provide these badges (https://jenkins.io/doc/pipeline/steps/badge/). Let me check with the team managing Jenkins to see if we can add that plugin, so that instead we can show the status again.

/cc @psftw @StefanScherer

Reply:
@thaJeztah Thanks! Sounds good. If you would like to provide the icon and status urls here, would be more than happy to update this pr to fix it.

Reply:
That'd be great; I'll await an answer from the test-infra team, and once the plugin is installed and functional, I'll post the correct URLs here 👍

Reply:
Plugin installed :tada:
The new badge URL should be `https://ci.docker.com/public/job/cli/job/master/badge/icon` and the corresponding job link URL should be `https://ci.docker.com/public/job/cli/job/master/`

Reply:
Awesome, thanks @psftw 🤗 🎉

Reply:
@psftw Thanks so much!